src/c++/perf_analyzer/request_rate_manager.cc

```diff
@@ -86,7 +86,7 @@ RequestRateManager::RequestRateManager(
 {
   if (on_sequence_model_) {
     for (uint64_t i = 0; i < num_of_sequences; i++) {
-      sequence_stat_.emplace_back(new SequenceStat(next_seq_id_++));
+      sequence_stat_.emplace_back(new SequenceStat(0));
```

Comment:
Why are all SequenceStat objects having their sequence ID initialized to 0? Won't that cause conflicts when `--num-sequences` is greater than 1? i.e. server will receive requests intended for different sequence IDs, but all the requests will be claiming to be for sequence ID 0.

Reply:
All of the sequences start inactive.  As soon as we go to issue a request for any of these, we will initialize a new sequence with a new sequence ID.  Allocating sequence IDs to these inactive sequences just eats up IDs and causes us to start from a weird ID (6) by default instead of 1.

Note that I made this change, and then realized that the parallel code in concurrency manager does the exact same thing. This code should probably just move to the parent LoadManager:
https://github.com/triton-inference-server/client/blob/main/src/c%2B%2B/perf_analyzer/concurrency_manager.cc#L89

Reply:
Nevermind, I see that it's re-initialized here:

https://github.com/triton-inference-server/client/blob/tgerdes-test-request-rate-sequences/src/c%2B%2B/perf_analyzer/load_manager.cc#L838